### PR TITLE
Synchronize manual with stubs for PHP 8.4 - part 5

### PR DESCRIPTION
**Pull request by kocsismate**
DOM class synopsis related changes.

### CODE REVIEW

File: reference/dom/domnamespacenode/wakeup.xml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="domnamespacenode.wakeup" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>DOMNode::__wakeup</refname>
+  <refpurpose>Forbids serialization unless serialization methods are implemented in a subclass</refpurpose>
```
Comment (nielsdos) [inline reference/dom/domnamespacenode/wakeup.xml:5]:
```suggestion
  <refpurpose>Forbids unserialization unless unserialization methods are implemented in a subclass</refpurpose>
```

File: reference/dom/domnamespacenode/wakeup.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="DOMNameSpaceNode">
+   <modifier>public</modifier> <type>void</type><methodname>DOMNameSpaceNode::__wakeup</methodname>
+   <void/>
+  </methodsynopsis>
+  <simpara>
+   Forbids serialization unless serialization methods are implemented in a subclass.
```
Comment (nielsdos) [inline reference/dom/domnamespacenode/wakeup.xml:15]:
```suggestion
   Forbids unserialization unless unserialization methods are implemented in a subclass.
```

File: reference/dom/domnode/wakeup.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<refentry xml:id="domnode.wakeup" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
+ <refnamediv>
+  <refname>DOMNode::__wakeup</refname>
+  <refpurpose>Forbids serialization unless serialization methods are implemented in a subclass</refpurpose>
```
Comment (nielsdos) [inline reference/dom/domnode/wakeup.xml:5]:
```suggestion
  <refpurpose>Forbids unserialization unless unserialization methods are implemented in a subclass</refpurpose>
```

File: reference/dom/domnode/wakeup.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+ </refnamediv>
+
+ <refsect1 role="description">
+  &reftitle.description;
+  <methodsynopsis role="DOMNode">
+   <modifier>public</modifier> <type>void</type><methodname>DOMNode::__wakeup</methodname>
+   <void/>
+  </methodsynopsis>
+  <simpara>
+   Forbids serialization unless serialization methods are implemented in a subclass.
```
Comment (nielsdos) [inline reference/dom/domnode/wakeup.xml:15]:
```suggestion
   Forbids unserialization unless unserialization methods are implemented in a subclass.
```

**Review by nielsdos — CHANGES_REQUESTED** [review]
Should also get entries in versions.xml. I implemented this in 8.1 and above.
The backstory here is that people used to have custom serialization methods on these classes, but that broke when someone added the ZEND_ACC_NOT_SERIALIZABLE flag to these classes, making it impossible to serialize even for child classes. So I came up with this trick to unbreak backwards compatibility.

**Comment by nielsdos** [comment]
> XPointer evaluation failed: #xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.domnamespacenode')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[@role='DOMNameSpaceNode'])

Looks like it broke CI

**Comment by kocsismate** [comment]
> Looks like it broke CI

Yeah, I'm currently debugging what the issue is ^^ I'm also going to fixup the version in versions.xml in the same commit.

**Comment by kocsismate** [comment]
Finally, I found it!

**Review by nielsdos — APPROVED** [review]
Thanks!

**Pull request merged**